utilities/src/main/java/io/syndesis/qe/bdd/CommonSteps.java

```diff
@@ -100,10 +100,15 @@ public void verifyBuild(String integrationName, int duration) {
     }
 
     @Given("^deploy AMQ broker$")
-    public void deployAMQBroker() throws Throwable {
+    public void deployAMQBroker() {
         AmqTemplate.deploy();
     }
 
+    @Given("^add \"([^\"]*)\" account$")
+    public void addAMQAccount(String protocol) {
```

Comment:
Do we really need a dedicated step to add account? It could be a parameter of `deployAMQBroker`. But I'd prefer to keep it simple and just add both to our `AccountDirectory` and the scenario will just create required connection via `Given created connections` step.

Reply:
@dsimansk I did it for a case with 2 different AMQ connections(openwire/amqp, e.g.) in the same integration , so that you need not to deploy amq server twice.

Reply:
@dsimansk - done.